src/js/count.js

```diff
@@ -8,6 +8,8 @@ class Count {
 	constructor (countEl, opts) {
```

Comment:
If we default opts we wouldn't need to constantly check it exists before using a property of it.
```suggestion
	constructor (countEl, opts = {}) {
```